awx/main/scheduler/dag_simple.py

```diff
@@ -89,8 +89,8 @@ def run_status(obj):
                 run_status(n['node_object']),
                 color
             )
-        for label, edges in self.node_from_edges_by_label.iteritems():
-            for from_node, to_nodes in edges.iteritems():
+        for label, edges in self.node_from_edges_by_label.items():
+            for from_node, to_nodes in edges.items():
```

Comment:
was there any issue that caused you to switch to items from iteritems? I would think iteritems should be preferred if not.

Reply:
iteritems is soo python 2.7

Reply:
oh right